### ISSUE TITLE
QGIS crashing on launch on Mac OS Catalina with libQt5Widgets.5.dylib not found error

### ISSUE THREAD

**Issue by owenlamont** [qgis/QGIS#32227]
**Describe the bug**
Installed QGIS 3.8.3 from the qgis-macos-pr.dmg file on Catalina. QGIS failed to launch with an error mentioning ibQt5Widgets.5.dylib was not found.

**How to Reproduce**

Tried to launch QGIS after installation on Catalina (I installed QGIS just after upgrading to Catalina).

**QGIS and OS versions**

QGIS 3.8.3
Mac OS X 10.15 (19A583)

**Additional context**

Here's a copy paste from the OS error dialog:
Process:               QGIS [2355]
Path:                  /Users/USER/*/QGIS.app/Contents/MacOS/QGIS
Identifier:            org.qgis.qgis3
Version:               3.8.3 (3.8.3 [exported])
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           QGIS [2355]
User ID:               501

Date/Time:             2019-10-13 11:10:00.437 +0800
OS Version:            Mac OS X 10.15 (19A583)
Report Version:        12
Bridge OS Version:     4.0 (17P572)
Anonymous UUID:        84C2A93B-71C2-EDF1-BD6C-AE0CF3ADF7D2

Time Awake Since Boot: 5200 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/libQt5Widgets.5.dylib
  Referenced from: /Users/USER/*/QGIS.app/Contents/MacOS/QGIS
  Reason: image not found

Binary Images:
       0x1069be000 -        0x106decff7 +org.qgis.qgis3 (3.8.3 - 3.8.3 [exported]) <7BC359A7-7C81-3235-8B76-CF3661983B41> /Users/USER/*/QGIS.app/Contents/MacOS/QGIS
       0x106dfa000 -        0x107671ff7 +libqgis_app.3.8.3.dylib (0) <FE02D51D-BA0D-3B0B-BE1B-1291E93D187E> /Users/USER/*/QGIS.app/Contents/MacOS/lib/libqgis_app.3.8.3.dylib
       0x107a56000 -        0x1081a1fff +org.qgis.qgis3_gui (3.8.3 - qgis_gui 3.8.3) <B69D3181-9D82-331A-94DB-273F67D162CC> /Users/USER/*/QGIS.app/Contents/Frameworks/qgis_gui.framework/Versions/3.8/qgis_gui
       0x1085fc000 -        0x1093d9fff +org.qgis.qgis3_core (3.8.3 - qgis_core 3.8.3) <1985B7C6-11EF-3D60-A889-486C3CC540F9> /Users/USER/*/QGIS.app/Contents/Frameworks/qgis_core.framework/Versions/3.8/qgis_core
       0x10f356000 -        0x10f3e493f  dyld (732.8) <42C11B81-6928-369F-B03E-D57355572700> /usr/lib/dyld

Model: MacBookPro15,1, BootROM 1037.0.78.0.0 (iBridge: 17.16.10572.0.0,0), 6 processors, 6-Core Intel Core i7, 2.6 GHz, 16 GB, SMC 
Graphics: kHW_IntelUHDGraphics630Item, Intel UHD Graphics 630, spdisplays_builtin
Graphics: kHW_AMDRadeonPro560XItem, Radeon Pro 560X, spdisplays_pcie_device, 4 GB
Memory Module: BANK 0/ChannelA-DIMM0, 8 GB, DDR4, 2400 MHz, Micron, 8ATF1G64HZ-2G6E1
Memory Module: BANK 2/ChannelB-DIMM0, 8 GB, DDR4, 2400 MHz, Micron, 8ATF1G64HZ-2G6E1
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x7BF), wl0: Jul 24 2019 00:02:16 version 9.30.357.3.32.5.33 FWID 01-22b80374
Bluetooth: Version 7.0.0f8, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
USB Device: My Passport 25E4
USB Device: My Passport 25E4
USB Device: USB 3.1 Bus
USB Device: Apple T2 Bus
USB Device: Touch Bar Backlight
USB Device: Touch Bar Display
USB Device: Apple Internal Keyboard / Trackpad
USB Device: Headset
USB Device: Ambient Light Sensor
USB Device: FaceTime HD Camera (Built-in)
USB Device: Apple T2 Controller
Thunderbolt Bus: MacBook Pro, Apple Inc., 47.3
Thunderbolt Bus: MacBook Pro, Apple Inc., 47.3

**Comment by PeterPetrik** [comment]
Hi @owenlamont . 
Is this all-in-one installer or the alternative installer?
Thanks.

**Comment by owenlamont** [comment]
Hi Peter, it was the all in one installer.

**Comment by PeterPetrik** [comment]
i am wondering because the location is not standard `/Users/USER/*/QGIS.app/Contents/MacOS/QGIS` .Where is your QGIS installed?

**Comment by owenlamont** [comment]
Actually I just reinstalled (I had uninstalled previously). It is now in /Applications/QGIS3.8.app

I realise what I've done now. Sorry this is a stupid mistake. I had installed the conda version of QGIS which was working fine for me. Some of the other devs recommended I install the all in one installer so I tried that without removing the conda install - I then was Command - Space launching QGIS and I think it was selecting the conda installed version before the dmg install which won't launch that way because the conda environment hadn't been activated. I checked I was launching the dmg install now and it is working fine.

Sorry for wasting your time - feel free to close this issue.
